pymeasure/instruments/eurotest/eurotestHPP120256.py

```diff
@@ -0,0 +1,320 @@
+        )
+    )
+
+    enable_kill = Instrument.setting(
+        "KILL,%s",
+        """ Enables or disables the kill function of the HV source.
+         When Kill is enabled yellow led is flashing and the output
+         will be shut OFF permanently without ramp if Iout > IOUTmax.""",
+        validator=strict_discrete_set,
+        values={'ON': 'ENable', 'OFF': 'DISable', 'ENable': 'ENable', 'DISable': 'DISable',
+                'EN': 'ENable', 'DIS': 'DISable'},
+        map_values=True
+    )
+
+    enable_output = Instrument.setting(
+        "HV,%s",
+        """Enables or disables the voltage output function of the HV source.
+         When output voltage is enabled green led is ON and the
+         voltage_setting will be present on the output""",
+        validator=strict_discrete_set,
+        values={'ON': 'ON', 'OFF': 'OFF', 'ENable': 'ON', 'DISable': 'OFF',
+                'EN': 'ON', 'DIS': 'OFF'},
+        map_values=True
+    )
+
+    @property
+    def id(self):
+        """ Returns the identification of the instrument """
+        log.info("Requesting instrument identification...")
+
+        response = self.ask("ID")
+        return response.encode(self.response_encoding).decode('utf-8', 'ignore')
+
+    def status(self):
+        """ Returns the unit Status which is a 16bits response where
+        every bit indicates teh state of one subsystem of the HV Source."""
+        log.info("Requesting instrument status...")
+
+        response = self.ask("STATUS,DI")
+        response = response.encode(self.response_encoding).decode('utf-8', 'ignore')
+        log.debug(f"Instrument status = {response}")
+        return response
+
+    def lam_status(self):
+        """ Returns the LAM status which is the status of the untit from the point
+        of view of the process. Fo example, as a response of asking STATUS,LAM, the HV
+        voltage could response one of the messages from the next list:
+        LAM,ERROR External Inhibit occurred during Kill enable
+        LAM,INHIBIT External Inhibit occurred
+        LAM,TRIP ERROR Software current trip occurred
+        LAM,INPUT ERROR Wrong command received
+        LAM,OK Status OK"""
+        log.info("Requesting instrument STATUS LAM...")
+
+        response = self.ask("STATUS,LAM")
+        response = response.encode(self.response_encoding).decode('utf-8', 'ignore')
+        log.debug(f"Instrument STATUS LAM = {response}")
+        return response
+
+    def emergency_off(self):
+        """ The output of the HV source will be switched OFF permanently and the values
+        of the voltage a current settings set to zero"""
+        log.info("Sending emergency off command to the instrument.")
+
+        self.write("EMCY OFF")
+        time.sleep(self.COMMAND_DELAY)
+
+    def shutdown(self, ramp):
+        """
+        Ramps the HV source to zero with a determinated ramp and without waiting
+        to the output reaches zero volts dissables it.
```

Comment:
I do not understand the comment: "without waiting to the output reaches zero volts disables it" (btw: disables has a single 's' in the middle).

Reply:
Yes, you are right.
done in commit. I hope now it is clearer.

Reply:
Yes, now I get it. It is not necessary to double the information in the general description and in the parameter description, but not I understand how to use the method.